src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -29,42 +29,84 @@ def _reindex_lake_to_link_id(target_df, crosswalk):
 
     # evaluate intersection of lake ids and target_df index values
     # i.e. what are the index positions of lake ids that need replacing?
-    lakeids = np.fromiter(crosswalk.keys(), dtype = int)
+    lakeids = np.fromiter(crosswalk.keys(), dtype=int)
     idxs = target_df.index.to_numpy()
     lake_index_intersect = np.intersect1d(
-        idxs, 
-        lakeids, 
-        return_indices = True
+        idxs,
+        lakeids,
+        return_indices=True
     )
 
     # replace lake ids with link IDs in the target_df index array
-    linkids = np.fromiter(crosswalk.values(), dtype = int)
+    linkids = np.fromiter(crosswalk.values(), dtype=int)
     idxs[lake_index_intersect[1]] = linkids[lake_index_intersect[2]]
 
     # (re) set the target_df index
-    target_df.set_index(idxs, inplace = True)
-    
+    target_df.set_index(idxs, inplace=True)
+
     return target_df
 
+
+def _parquet_output_format_converter(df, start_datetime, dt, configuration):
+    '''
+    Utility function for convert flowveldepth dataframe
+    to a timeseries and to match parquet input format
+    of TEEHR
+
+    Arguments:
+    ----------
+    - df (DataFrame): Data frame to be converted
+    - start_datetime: Date time from restart parameters
+    - dt: Time step
+    - configuration: configuration (for instance- short_range, medium_range)
+
+    Returns:
+    --------
+    - timeseries_df (DataFrame): Converted timeseries data frame
+    '''
+
+    df.index.name = 'location_id'
+    df.reset_index(inplace=True)
+    timeseries_df = df.melt(id_vars=['location_id'], var_name='var')
```

Comment:
I think if you use `value_vars=[ 'q', 'v', 'd' ]` as a kwarg to melt, you might have an easier time extracting from un-pivoted table?

The below method seems like there should be a better way besides casting to string, manipulating the string, and recasting to numeric/datetime types. 

I'm not sure exactly what the `df` looks like that is trying to be manipulated at this point, but I would try to consider a different method(s) for extracting the needed data from it.

If for some reason this is the only way, then please comment this implementation to describe what the state of the df is and why this is the way it needs to be manipulated.

Reply:
I looked into options and this seems to be the only way. I had to manipulate the strings and cannot use `value_vars=[ 'q', 'v', 'd' ]` as a kwarg to melt because of the `df` format. Please look at the screenshot below.

<img width="1127" alt="parquet t-route" src="https://github.com/NOAA-OWP/t-route/assets/22774612/b1704dba-6f72-409d-b4dd-62c3377bbfd0">

Each column name consists of a time step and a variable name (`q`, `v` or `d`) in string format. The time steps values are used to get the value_time.